test/still/compiler/incremental/node_test.exs

```diff
@@ -1,19 +1,11 @@
 defmodule Still.Compiler.Incremental.NodeTest do
-  use Still.Case
+  use Still.Case, async: false
```

Comment:
Why ensure it to be `false`?

Reply:
There are a few stochastic tests because of processes (Node) already being started. This helps prevents those, but I know we'll need a better system to scope processes, or file names, to test files.

Reply:
Ugh, I hate that. Can we document the reason for test failures somewhere. Even if it's a ticket on Trello or an issue on GitHub?

Reply:
Well, the reason is the way we designed the system. We need to do some work to make the whole thing easier to test. I'll add a card.